roles/confluent.zookeeper/tasks/health_check.yml

```diff
@@ -5,6 +5,8 @@
   shell: "{{zookeeper_health_check_command}}"
   args:
     executable: /bin/bash
+  environment:
```

Comment:
so its been a while since I've used this feature, but we have this `proxy_env` variable, thats added as an env var to all of our plays:
```
- name: Kafka Broker Status Finding
  hosts: kafka_broker
  gather_facts: false
  tags: kafka_broker
  environment: "{{ proxy_env }}"
```

Initially it was designed for just proxy env vars:
```
    #### Setting Proxy Environment variables ####
    ## To set proxy env vars for the duration of playbook run, uncomment below block and set as necessary
    # proxy_env:
    #   http_proxy: http://proxy.example.com:8080
    #   https_proxy: http://proxy.example.com:8080
```

In your case you want to set more env vars (ie JAVA_HOME), so first step is try adding
```
proxy_env:
  JAVA_HOME: /usr/local/confluent/java
```

To you inventory, lets see if that works, and then we may need to think about renaming the proxy_env var or something